patches/chrome-browser-ui-webui-realbox-realbox_handler.cc.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/ui/webui/realbox/realbox_handler.cc b/chrome/browser/ui/webui/realbox/realbox_handler.cc
+index f2bc9c8997b3a..fac7fa689b576 100644
+--- a/chrome/browser/ui/webui/realbox/realbox_handler.cc
++++ b/chrome/browser/ui/webui/realbox/realbox_handler.cc
+@@ -457,7 +457,7 @@ std::string GetBase64UrlVariations(Profile* profile) {
+ 
+ // TODO(crbug.com/1431513): Consider inheriting from `ChromeOmniboxClient`
+ //  to avoid reimplementation of methods like `OnBookmarkLaunched`.
+-class RealboxOmniboxClient final : public OmniboxClient {
++class RealboxOmniboxClient final : public BraveOmniboxClient {
```

Comment:
I couldn't work out a way to do this without patching, but if anyone has ideas, I'm listening :D

Reply:
If we don't have it we crash - we cast an `OmniboxClient` to a `BraveOmniboxClient` in `brave/chromium_src/components/omnibox/browser/omnibox_controller.cc`

Reply:
Good call! I've deleted `BraveOmniboxClient` and removed the relevant patches :smile: